scargo/commands/update.py

```diff
@@ -95,6 +95,26 @@ def scargo_update(config_file_path: Path) -> None:
 
     if project_config.build_env == SCARGO_DOCKER_ENV:
         if not Path(project_path, ".dockerenv").exists():
-            scargo_docker_build([])
+            pulled_image = False
```

Comment:
this is not the best place in my opinion - build should always build the image on demand, 
we want to pull docker image only for scargo update and if image not exist than build

Reply:
And what makes you think that build will not always build the image on demand?